dispatcher/src/main/kotlin/pl/droidsonroids/testing/mockwebserver/condition/HttpUrlCondition.kt

```diff
@@ -0,0 +1,23 @@
+package pl.droidsonroids.testing.mockwebserver.condition
+
+import okhttp3.HttpUrl
+import okhttp3.mockwebserver.RecordedRequest
+
+/**
+ * A Condition which matches HttpUrls only, they are extracted from request so that only
+ * <code>isUrlMatching(HttpUrl)</code> needs to implemented.
```

Comment:
bardziej czytelniejsze jest chyba `{@code ...}` od `<code>...</code>`
https://stackoverflow.com/questions/7066409/what-is-the-difference-between-code-memberdata-and-codememberdata-code-in